backend/src/services/audit.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../models');
+
+module.exports = {
+  async findHistory(acronym) {
+    return await db.Acronym.findOne({
+      where: { acronym: acronym },
```

Comment:
Might not be a bad idea to have this where be after the include attribute (alphabetical ordering convention seen in other models)

Reply:
Yes, fixed